zokrates_js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "zokrates-js",
+  "main": "js/index.js",
+  "author": "Darko Macesic <darem966@gmail.com>, Edi Sinovcic <edi@blockchain-it.hr>",
+  "version": "1.0.17",
```

Comment:
do we want to keep versions synced?

Reply:
I think it's not needed, but we can discuss this further.

Reply:
added to todo